.gitmodules

```diff
@@ -391,3 +391,6 @@
 [submodule "libraries/drivers/at24mac_eeprom"]
 	path = libraries/drivers/at24mac_eeprom
 	url = https://github.com/facts-engineering/CircuitPython_AT24MAC_EEPROM.git
+[submodule "libraries/drivers/ruhrohrotaryio"]
+	path = libraries/drivers/ruhrohrotaryio
+	url = git@github.com:todbot/CircuitPython_Community_Bundle.git
```

Comment:
I think the url here needs to be `https://` instead of `git@` and this URL is pointing back to a fork of the Bundle rather than then new sub-project. 

I think this URL should be:
```
https://github.com/todbot/CircuitPython_RuhRohRotaryIO.git
```